http/server.lua

```diff
@@ -189,11 +192,12 @@ local function listen(tbl)
 	end
 	local ctx = tbl.ctx
 	if ctx == nil and tls ~= false then
```

Comment:
Perhaps error if `tls == false` and using a unix socket? I don't think there is a sensible default for the canonical name in the certificate.